client/db.go

```diff
@@ -158,6 +158,9 @@ func Open(addr string) (*DB, error) {
 	if err != nil {
 		return nil, err
 	}
+	if u.User == nil {
+		u.User = url.User("root")
```

Comment:
Both "root" and `user.Current()` sound equally correct to me. But I thought hardcoding it would at least save `client.Open` an additional syscall for now.